.github/workflows/spike-openocd-tests.yml

```diff
@@ -12,65 +19,99 @@ on: pull_request
 jobs:
   test:
     name: Test debug (Ubuntu)
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
I prefer latest. If we run against latest and it breaks, we can quickly fix it by changing back to the previous version. If the fixed version stops working because github removes it (as just happened for ubuntu-18.04) then the only way to quick fix is to move to a newer version and hope that it works. If it doesn't we have to make the test work with newer ubuntu right now.

Reply:
I usually prefer to be explicit about requirements/dependencies, that's why I proposed `ubuntu-22.04` (and `ubuntu-latest` is at the moment just an alias to `ubuntu-22.04`). But it is in fact not that important - I reverted it back to `ubuntu-latest`.